### ISSUE TITLE
Image alt text missing from SDK

### ISSUE THREAD

**Issue by sundeqvist** [makenotion/notion-sdk-js#517]
Hey! 👋 

Image alt texts that are added in Notion are not parsed by the SDK. It looks like alt texts are a relatively new feature in Notion - are there plans to add support to the SDK?

I have a workaround where I use the caption field, but as I have other data in the caption it's quickly turning into a mess 😭 

Thanks!

**Comment by alexanderkalachev** [comment]
I hope it will be added too

**Comment by vojtabohm** [comment]
How is this issue looking?

**Comment by ArthurRM** [comment]
Yes, this is needed!
